chain/chain-primitives/src/error.rs

```diff
@@ -304,6 +304,70 @@ impl Error {
             _ => false,
         }
     }
+
+    /// Some blockchain errors are reported in the prometheus metrics. In such cases a report might
+    /// contain a label that specifies the type of error that has occured. For example when the node
+    /// receives a block with missing chunks this would be reported as:
```

Comment:
This happens to be a bad example as a block with missing chunks shouldn't be considered invalid. I'm guessing the error actually means something a little bit different. Can you fix it or just change to a more obvious error like invalid signature?

Reply:
I picked "chunks_missing" because it was one of the variants found in the wild (https://github.com/near/nearcore/pull/10164#issuecomment-1809335895).

If a block with missing chunks isn't invalid, then maybe it's wrong to increase the metric when we run into such blocks (?)

I will change it to invalid signature, but I'm a bit worried why `chunks missing` increases the number of invalid_blocks when it isn't strictly an invalid block.

Reply:
Chunks missing may have some slightly different meaning such as the block has headers for all chunks but the node hasn't received all of the chunks. We would need to dig a bit deeper to understand under what circumstances it happens - feel free to check it out. My guess is that it is a legit error but just slightly confusing name.